docs/how-tos/howto-query-operator.md

```diff
@@ -33,7 +33,7 @@ This will return a list of libp2p addresses that are currently peers of the runn
 
 For the status of a specific peer, run:
 ````bash
-go run main.go peers status <address>
+go run main.go peers status --libp2p-node-id <address>
```

Comment:
I guess this links back to the [same question ](https://github.com/0xPolygon/polygon-sdk/pull/61#discussion_r647265538) on the original PR